src/libxrpl/protocol/STTx.cpp

```diff
@@ -187,6 +187,17 @@ STTx::getSeqProxy() const
     if (seq != 0)
         return SeqProxy::sequence(seq);
 
+    if (isFieldPresent(sfBatchTxn))
+    {
+        STObject const batchTxn = const_cast<ripple::STTx&>(*this)
+                                      .getField(sfBatchTxn)
+                                      .downcast<STObject>();
+        std::uint32_t const startSequence{
```

Comment:
🤔  If it is a multi-account transactions, according to the [SPEC](https://github.com/XRPLF/XRPL-Standards/discussions/162) 4.1 
`
Multi-account transactions use the same "phantom sequence number" strategy, but instead uses the equation BatchTxn.Sequence + BatchTxn.BatchIndex, since the OuterSequence does not map to the inner transaction's account.` 
Should the startSequence be the BatchTxn.Sequence for multiple account inner transactions?

Reply:
I'm just using `OuterSequence` the same. I thought 2 sequences was confusing. I can update it.

Reply:
@dangell7 and I discussed this separately and the spec will be updated. We're somewhat unsure about which is more confusing, but this can always be changed later after users have a chance to try it out.